### PR TITLE
UI: Add DAS namespaces toggle and fix empty card bug

### PR DESCRIPTION
**Pull request by backspace**
This builds on filtering to allow the optimize page to show recommendations
for the active namespace vs all namespaces. If turning off the toggle causes
the summary from the active card to become excluded from the filtered list,
the active summary changes, as with the facets.

It also includes a fix for the bug from [here](https://github.com/hashicorp/nomad/pull/9294#pullrequestreview-527748994). I’ll address the facet duplication in a followup 🤓

### CODE REVIEW

File: ui/app/styles/components/toggle.scss [hunk 1]
```diff
@@ -10,6 +10,16 @@ $size: 12px;
...
+  &.is-flex {
+    display: flex;
+    align-items: center;
+
+    .label {
+      padding-left: 0.5rem;
+      padding-bottom: 1px;
+    }
+  }
+
```
Comment (backspace) [inline ui/app/styles/components/toggle.scss:22]: I’m not sure how acceptable this is… there’s nowhere to really target this toggle specifically but maybe it needs to be its own component Sass file 😞

File: ui/app/styles/components/toolbar.scss [hunk 2]
```diff
@@ -8,6 +8,10 @@ $spacing: 1.5em;
   margin-left: -$spacing / 4;
   margin-right: -$spacing / 4;
 
+  &.collapse + .toolbar {
+    margin-top: -$spacing / 2;
+  }
+
```
Comment (backspace) [inline ui/app/styles/components/toolbar.scss:14]: I placed the toggle in a second `toolbar` and use this to bring them closer together. Maybe there’s a better approach? 🤔

File: ui/app/styles/components/toggle.scss (hunk 1 above)
Comment (DingoEatingFuzz) [reply ui/app/styles/components/toggle.scss:22]: Yeah, there isn't a current better alternative. Ideally these types of variants/exceptions get cleaned up more systemically when we rip out Bulma.

File: ui/app/styles/components/toolbar.scss (hunk 2 above)
Comment (DingoEatingFuzz) [reply ui/app/styles/components/toolbar.scss:14]: This is sorta in the same category as the above comment. No better approach currently. I like that this is targeted specifically at two toolbars with the collapse class though.

**Review by DingoEatingFuzz — APPROVED** [review]

**Pull request merged**